Automattic/wp-calypso
# Stats: Refactor component StatsViews and its styles

#### Proposed Changes

* Delete the unused component `StatsViews`.
* Refactor remaining `StatsViewsMonths` styles into existing shared styles.

#### Testing Instructions



* Spin up this change with the Calypso Live link.
* Navigate to the Stats `Insights` page.
* Ensure the All-time insights table styles work as previously.
* Navigate to the Stats `Post Detail` page with the feature flag `/stats/post/${post-id}/${site-id}?flags=stats/enhance-post-detail`.
* Ensure the All-time Insights table styles work as previously.

#### Pre-merge Checklist





Related to #71762
